src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String SEPARATOR = "\n";
+    private static final String PREFIX = "fruit,quantity\n";
+    private static final String SUFFIX = "\n";

```

Comment:
The SUFFIX constant is redundant since you're already using SEPARATOR at the end of the stream. You can remove it to simplify the code.